sui-execution/latest/sui-verifier/src/private_generics.rs

```diff
@@ -160,8 +161,12 @@ fn verify_private_event_emit(
     type_arguments: &[SignatureToken],
 ) -> Result<(), String> {
     let fident = view.identifier_at(fhandle.name);
+    if fident == GET_EVENTS_TEST_FUNCTION {
```

Comment:
Might be worthwhile to have something like `PUBLIC_EVENT_FUNCTIONS` to match the other private generics code, but this is probably fine as is